File: Api/Api/Domain/Model/Person.cs
using Api.Domain.Model.Base;

namespace Api.Domain.Model
{
    public class Person : BaseTable
    {
        public virtual int Id { get; protected set; }
        public virtual string Name { get; set; }
        public virtual string Department { get; set; }
        public virtual string Email { get; set; }
        public virtual string Phone { get; set; }
    }
}

File: App/lib/person/person.dart
class Person {
  String? id;
  String name;
  String? department;
  String? phone;
  String email;

  Person(
      {this.id,
      required this.name,
      this.department,
      this.phone,
      required this.email});

  static Person fromMap(Map<String, dynamic> map) {
    return Person(id: map['id'], name: map['name'], department: map['department'], phone: map['phone'], email: map['email']);
  }
}